gocwebtemplate-sample-jsp/src/main/java/goc/webtemplate/jsp/samplebeans/FeedbackAndShareThisPageSampleBean.java

```diff
@@ -1,17 +1,31 @@
 package goc.webtemplate.jsp.samplebeans;
 
-import goc.webtemplate.Constants;
+import java.util.ArrayList;
+import java.util.List;
 
+import goc.webtemplate.Constants;
+import goc.webtemplate.FeedbackLink;
 import goc.webtemplate.component.jsp.DefaultTemplateCoreBean;
 
 public class FeedbackAndShareThisPageSampleBean extends DefaultTemplateCoreBean {
 
     @Override
     public void onWebTemplateInitialize() {
-        this.setShowFeedbackLink(true);
-        this.setFeedbackUrl("http://www.google.ca"); 
-        this.setFeedbackUrlFr("http://www.google.ca/?hl=fr");
+        List<String> elements = new ArrayList<String>();

```

Comment:
- Put a comment on top to explain why we're doing this (can copy the one line from the sample page)
- Make sure french department is not corrupted
- (also applies to spring sample)

Reply:
done